data_analysis/README.md

```diff
@@ -0,0 +1,54 @@
+            * `data_date`: Date the data was scraped
+            * `data_hour`: Hour extracted from data timestamp	
+            * `rt`: Route (from original API response; see API documentation)	
+            * `des`: Direction (from original API response; see API documentation)	
+            * `vid`: A set containing the unique `vid` values observed during the given data hour on the given `rt` with the given `des` value (`vid` values come from original API resonse; see API documentation)	
+            * `tatripid`: A set containing the unique `tatripid` values observed during the given data hour on the given `rt` with the given `des` value (`tatripid` values come from original API resonse; see API documentation)
+            * `tablockid`: A set containing the unique `tablockid` values observed during the given data hour on the given `rt` with the given `des` value (`tablockid` values come from original API resonse; see API documentation)
+            * `vh_count`: Length of `vid` set (i.e., count of unique `vid` values encountered during this data hour on the given `rt` with the given `des` value)
+            * `trip_count`: Length of `tatripid` set (i.e., count of unique `tatripid` values encountered during this data hour on the given `rt` with the given `des` value)
+            * `block_count`: Length of `tablockid` set (i.e., count of unique `tablockid` values encountered during this data hour on the given `rt` with the given `des` value)
+        * In S3, these are available in the `chn-ghost-buses-public` bucket in a folder called `bus_hourly_summary_v2`. Within that folder there is one file per full day from `2022-05-20` until `2022-08-07`. So, for example, `pandas.read_csv('s3://chn-ghost-buses-public/bus_hourly_summary_v2/2022-05-20.csv')` should work to load the hourly summaries from May 20.
+* **schedule_summaries**: To compare the realtime data with the schedule, [`static_gtfs_analysis.ipynb`](https://github.com/chihacknight/chn-ghost-buses/blob/main/data_analysis/static_gtfs_analysis.ipynb) ingests a GTFS schedule file and outputs a file that aggregates to a count of unique trips per hour on a given route. There are two flavors:
+    * `route_level`: 
+        * These files do not distinguish between different route directions/destinations (so Howard-bound #22 Clark bus trips are combined with Harrison-bound #22 Clark bus trips). Their schema is:
+            * `date`: Schedule date (can be joined with `bus_hourly_summaries.data_date`)
+            * `route_id`: Route identifier (can be joined with `bus_hourly_summaries.rt`)
+            * `hour`: Schedule hour (can be joined with `bus_hourly_summaries.data_hour`)
+            * `trip_count`: Count of unique trip IDs scheduled to have any scheduled stops within the given hour on the given route (can be compared with `bus_hourly_summaries.trip_count`)
```

Comment:
do trip id's count unique "trips" which are when a bus completes a route in one direction?

Reply:
Trip IDs *should* correspond to scheduled GTFS information: https://gtfs.org/schedule/reference/#tripstxt. So broadly, yes, but there can be some nuances / quirks